common/src/Assets/AttributeDefinition.cpp

```diff
@@ -302,6 +325,18 @@ namespace TrenchBroom {
             m_options.push_back(FlagsAttributeOption(value, shortDescription, longDescription, isDefault));
         }
 
+        String FlagsAttributeDefinition::optionDescriptions() const {
+            StringStream stream;
+            for (auto& option : m_options) {
+                stream << option.value() << " = " << option.shortDescription();
```

Comment:
Maybe add some indentation or bullet points?

Reply:
OK if I make optionDescriptions() return wxString, so we can use bullets? It's a UI/presentation function so I guess it makes sense.

Reply:
I'd like to keep these dependencies to an absolute minimum. Maybe do the formatting in the UI layer then and get the descriptions as a list there?

Reply:
Agreed, sticking the UI formatting code here was just a lazy hack to avoid a double dispatch/vistor set up.. but on second thought that's probably cleaner, and will allow keeping the wxString dependency in the UI code.

Reply:
Wait, you can avoid double dispatch here as you can get the type of an attribute definition via `AttributeDefinition::Type`.

Reply:
So no need to do a double dispatch here.